src/engine/basics/Settings.hxx

```diff
@@ -158,6 +159,8 @@ struct SettingsData
 
   /// Write errors to a log file
   bool writeErrorLogFile;
+
+  std::map<std::string, uint32_t> fontSizeMap;
```

Comment:
Why is this a map?

Reply:
you can have much parametres for default font size, it associative map for it
for examaple "Button": 20, "Settings Menu": 30